### PR TITLE
Updated README file to reference library's version 2.

### PR DESCRIPTION
**Pull request by diogogmt**
I was having some issues with version 1 where the unix timestamp was missing the millisecond case when I happily realized that the bug was fixed as part of version 2 :)

### CODE REVIEW

**Comment by gravis** [comment]
Hi, the version 2 is not officially out :)
I'm currently on holidays, and will release the v2 next week :) (along with this PR)

File: README.md [hunk 1]
```diff
@@ -15,35 +15,36 @@ The hook must be configured with:
...
 import (
-    "log/syslog"
-    "github.com/Sirupsen/logrus"
-    "gopkg.in/gemnasium/logrus-graylog-hook.v1"
+    log "github.com/Sirupsen/logrus"
+    "gopkg.in/gemnasium/logrus-graylog-hook.v2"
     )
 
 func main() {
-    log := logrus.New()
```
Comment (gravis) [inline README.md]: Why did you remove this?

File: README.md [hunk 2]
```diff
@@ -15,35 +15,36 @@ The hook must be configured with:
...
-    hook := graylog.NewAsyncGraylogHook("<graylog_ip>:<graylog_port>", "some_facility", map[string]interface{}{"this": "is logged every time"})
+    hook := graylog.NewAsyncGraylogHook("<graylog_ip>:<graylog_port>", map[string]interface{}{"facility": "awesomeMicroService"})
     defer hook.Flush()
-    log.Hooks.Add(hook)
```
Comment (gravis) [inline README.md]: and this?

Comment (diogogmt) [inline README.md]: It is missing the ```log.AddHook(hook)``` statement.
I can add it back.

File: README.md (hunk 1 above)
Comment (diogogmt) [inline README.md]: I found that there is no need to create a new instance of logrus just for the graylog hook. Plus when creating a new instance it wouldn't allow to set the default level for the logs.

Is there a specific reason to create a new instance of logrus?
